packages/ui/src/components/filters/filters-bar/filter-variants/checkbox.tsx

```diff
@@ -50,19 +42,17 @@ const Checkbox = ({ filter, filterOption, onUpdateFilter, searchQueries, handleS
                   )
                 })}
 
-              <DropdownMenuItem className="p-0 focus:bg-transparent" asChild>
-                <Input
-                  className="h-6 flex-1 border-none outline-none hover:border-none focus:border-none focus-visible:ring-0"
-                  type="text"
-                  placeholder={filter.selectedValues.length > 0 ? '' : 'Select one or more options...'}
-                  value={searchQueries.filters[filter.type] || ''}
-                  onChange={e => handleSearchChange?.(filter.type, e.target.value, 'filters')}
-                  onClick={e => {
-                    e.preventDefault()
-                  }}
-                  onKeyDown={e => e.stopPropagation()}
-                />
-              </DropdownMenuItem>
+              <Input
+                className="h-6 flex-1 border-none outline-none hover:border-none focus:border-none focus-visible:ring-0"
+                type="text"
+                placeholder={filter.selectedValues.length > 0 ? '' : 'Select one or more options...'}
```

Comment:
Not an issue, but you can simplify a bit here.
```suggestion
                placeholder={filter.selectedValues.length ? '' : 'Select one or more options...'}
```